trunner/tools/phoenix.py

```diff
@@ -119,8 +125,11 @@ def _run(self):
             # such wait avoids a following issue with loading an app:
             # https://github.com/phoenix-rtos/phoenix-rtos-project/issues/545
             time.sleep(0.5)
-        except (TimeoutError, Exception) as exc:
+        except (TimeoutError, ValueError) as exc:
             raise PhoenixdError(str(exc)) from exc
+        except Exception as exc:
+            # add traceback if it's an internal exception
+            raise PhoenixdError(str(exc), traceback=traceback.format_exc()) from exc
```

Comment:
replacing exception traceback is kindof misleading (no PhoenixdErorr exception is raised at the line provided in traceback). My proposed solution provided real traceback (2 tracebacks because of `from` usage)